tests/cppunit/compact_test.cc

```diff
@@ -38,7 +38,7 @@ TEST(Compact, Filter) {
 
   int ret;
   std::string ns = "test_compact";
-  auto hash = new Redis::Hash(storage_.get(), ns);
+  auto hash = Util::MakeUnique<Redis::Hash>(Redis::Hash(storage_.get(), ns));
```

Comment:
```suggestion
  auto hash = Util::MakeUnique<Redis::Hash>(storage_.get(), ns);
```
And same as all of other changes.